### PR TITLE
News background list layout color

### PR DESCRIPTION
**Pull request by wobrak**
News background list layout  color and margin

### CODE REVIEW

File: TOZ_iOS/News.storyboard [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12086"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12088"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
         <!--NewsViewController-->
         <scene sceneID="Ic6-uI-xWc">
             <objects>
-                <viewController title="News" id="LDg-fo-A5n" userLabel="NewsViewController" customClass="NewsViewController" customModule="TOZ_iOS" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController title="News" automaticallyAdjustsScrollViewInsets="NO" id="LDg-fo-A5n" userLabel="NewsViewController" customClass="NewsViewController" customModule="TOZ_iOS" customModuleProvider="target" sceneMemberID="viewController">
```
Comment (piotr-tobolski) [inline TOZ_iOS/News.storyboard]: Is this option needed?

File: TOZ_iOS/News.storyboard [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
                     <layoutGuides>
                         <viewControllerLayoutGuide type="top" id="s1P-l8-cHA"/>
                         <viewControllerLayoutGuide type="bottom" id="nFv-2U-hRa"/>
                     </layoutGuides>
-                    <view key="view" contentMode="scaleToFill" id="OtT-de-cLq">
-                        <rect key="frame" x="0.0" y="0.0" width="414" height="736"/>
+                    <view key="view" contentMode="scaleToFill" misplaced="YES" id="OtT-de-cLq">
```
Comment (piotr-tobolski) [inline TOZ_iOS/News.storyboard]: misplaced?

File: TOZ_iOS/News.storyboard (hunk 1 above)
Comment (wobrak) [inline TOZ_iOS/News.storyboard]: After changes applied Extend Edges, this option doesn't  affect on bounce area so I can switch it off

File: TOZ_iOS/News.storyboard (hunk 2 above)
Comment (wobrak) [inline TOZ_iOS/News.storyboard]: Then again, automaticallyAdjustsScrollViewInsets is needed to set off,  it removes this

**Comment by wobrak** [comment]
Done

**Pull request merged**